cpp-client/README.md

```diff
@@ -27,7 +27,7 @@ C++ compiler and tool suite (cmake etc).
 6. Build and install dependencies for Deephaven C++ client.
 
    Decide on a directory for the dependencies to live (eg, "$HOME/dhcpp").
-   Create that directory and copy the `$DHSRC/deephaven-core/cpp-client/deephaven/client/build-dependencies.sh`
+   Create that directory and copy the `$DHSRC/deephaven-core/cpp-client/build-dependencies.sh`
```

Comment:
By the way, my own workflow for doing this is

* create that directory
* cd to that directory
* run $DHSRC/deephaven-core/cpp-client/build-dependencies.sh

Put another way, I never copy build-dependencies.sh over to my target directory. I simply set my working directory to the target and run build-dependencies.sh using its full path.

(I realize that in some cases people modify build-dependencies.sh, but even in that case, they can make their own decisions about modifying it and where the modified file should live)

Up to you, but I mention this because (depending on your opinion) it might simplify these instructions slightly, and it might be a *tad* more in line with other projects which don't leave a build script behind in the output directory.

Reply:
Devin wants me to remove build-dependencies.sh completely from this repo since it has to also live in https://github.com/deephaven/deephaven-base-images/blob/main/cpp-client/build-dependencies.sh.
So I've re-done the README and done some of what you asked (but the example still copies locally, since the script doesn't exist anymore in the clone of deephaven-core).  PTAL and see what you think.

Reply:
Ah, looks fine. Thank you. You can resolve this :-)